liferay-echo/liferay-portal
# LPS 197679 Show missing labels in DPT's infopanel and formatting

Motivation
=======
When reviewing the following [figma](https://www.figma.com/file/ydHldCxSGZuPe1tuHu9jXe/LPS-189856---Usability-improvements-for-objects-page-builder-integration?type=design&node-id=1-453&mode=design&t=ZOGpD9JjHbOKG9IM-0), we realized that some data and formatting was missing.

Proposed Solution
========
Implement the missing data and formatting when the data is shown.

Steps to Verify
========
1. Go to Design > Page Templates > Display Page Templates Tab.
2. Create a new Folder.
3. Create a new Display Page Template.
4.  Check that the styles and data showed in the infopanels matched the examples given in the previously linked figma file.

Commits
========
- LPS-197679 Show status and default label when corresponds in LayoutPageTemplateEntry info-panel
- LPS-197679 Unifying info-panel styles
- LPS-197679 Unifying info-panel date styles
- LPS-197679 Hiding description when empty
- LPS-197679 Add missing language key
- LPS-197679 Add outer div so it gets proper height
- LPS-197679 Fix spacing and text style
- LPS-197679 buildLang
